index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
       </div>
       <div class="col-7_md-12_md-first hero">
         <div>
-          <img class="img md-hidden" src="{{ "/images/hero.svg" | relative_url }}" />
-          <img class="img md-gt-hidden" src="{{ "/images/hero-mobile.svg" | relative_url }}" />
+          <img class="img" srcset="./images/hero.png 1x, ./images/hero@2x.png 2x, ./images/hero@3x.png 3x" alt="Crossplane hero image" />
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper